.gitignore

```diff
@@ -1,3 +1,4 @@
-node_modules
 build
-/lib
+coverage
+lib
```

Comment:
The `/lib` is necessary to target only the `lib` that is generated by the gulp build task (and not the one under `src`). Filed https://github.com/juttle/juttle-viz/issues/22 to address the issue of multiple `lib` folders.